### PR TITLE
Fix for python 3.8

### PR DESCRIPTION
**Pull request by omgitsraven**
See here: https://stackoverflow.com/questions/59014318/filenotfounderror-could-not-find-module-libvlc-dll

### CODE REVIEW

File: isrcsubmit.py [hunk 1]
```diff
@@ -43,6 +43,8 @@
 from optparse import OptionParser
 from subprocess import Popen, PIPE, call
 
+os.add_dll_directory(os.getcwd())
```
Comment (jesus2099) [inline isrcsubmit.py:46]: I am in Python 64-bit v3.7.4 in Windows 10, this row throws following error and prevents isrcsubmit from running:

    AttributeError: module 'os' has no attribute 'add_dll_directory'

What could we do to fix script for version 3.8 without breaking older versions?

Comment (jesus2099) [reply isrcsubmit.py:46]: Same with Python 32-bit v3.4.3 in Windows XP.

(both setups are running fine without this line)

**Comment by Freso** [comment]
No response from submitter in over a year. Going to close this. I’ll keep the issue open though and may look into a way to fix this. Thanks for the PR!

**Comment by jesus2099** [comment]
I found a way to run this fix only in Windows Python 3.8+, only when it is both required and non-breaking: #144.

**Pull request closed without merging**